src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -427,6 +445,52 @@ class CollabWrapper extends PureComponent<Props, CollabState> {
     this.excalidrawAPI.history.clear();
   };
 
+  private initializeIdleDetector = () => {
```

Comment:
We should remove the listeners (and potentially reset the timers) on unmount. Not an issue at this moment, but may become one in the future without us noticing.

Reply:
Done!